packages/pendle/src/Pendle.test.ts

```diff
@@ -0,0 +1,56 @@
+import { apply } from '@rabbitholegg/questdk/filter'
+import { describe, expect, test } from 'vitest'
+import { swap } from './Pendle.js'
+import {
+  SWAP_EXACT_PT_FOR_YT,
+  SWAP_EXACT_SY_FOR_YT,
+  SWAP_EXACT_TOKEN_FOR_YT,
+  SWAP_EXACT_YT_FOR_PT,
+  SWAP_EXACT_YT_FOR_SY,
+  SWAP_EXACT_YT_FOR_TOKEN,
+} from './test-transactions'
+import { getRouterAddress } from './contract-addresses.js'
+import { SUPPORTED_CHAINS } from './chain-ids.js'
+import { SwapYTV3_ABI } from './abi.js'
+
+describe('Given the Pendle plugin', () => {
+  describe('When handling the swap', () => {
+    const validTransactions = [SWAP_EXACT_TOKEN_FOR_YT, SWAP_EXACT_YT_FOR_SY]
+    const invalidTransactions = [
```

Comment:
Generally an invalid transaction would be a transaction that has params that don't align with the filter being applied - what about these transactions makes them invalid for this context?

Reply:
Pendle recently released Router V3, and these transactions were sent to the old contract addresses. So I think they should be considered as invalid transactions